### PR TITLE
GAWB-2474 fix regressions no matching entities in launch analysis ui

### PR DESCRIPTION
**Pull request by ahaessly**
fix (actually revert) method tests to work with method table

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by ahaessly** [comment]
the changes in GAWB-2104 actually fixed the regression issues. this ticket is just reverting the tests to what they used to be

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/methods/MethodConfigDetailsPage.scala [hunk 1]
```diff
@@ -216,9 +219,9 @@ class LaunchAnalysisModal(implicit webDriver: WebDriver) extends FireCloudView {
       click on (await enabled launchAnalysisButtonQuery)
     }
 
-    def getSelectedEntityText(): String = {
-      await enabled selectedEntityQuery
-      find(selectedEntityQuery).fold("")(_.text)
+    def isNoDefaultEntitiesMessagePresent(): Boolean = {
+      await enabled emptyDefaultEntitiesMessageQuery
```
Comment (breilly2) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/methods/MethodConfigDetailsPage.scala:223]: Probably don't need to wait for it to be enabled and also find it. Could probably just find (or findAll if you expect to have to wait... see Scalatest or Selenium docs... I don't have a good link handy).

**Review by breilly2 — APPROVED** [review]
:+1: Only one minor suggestion.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/methods/MethodConfigDetailsPage.scala (hunk 1 above)
Comment (ahaessly) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/methods/MethodConfigDetailsPage.scala:223]: i needed the wait there.

**Comment by ahaessly** [comment]
this actually did not make it onto dev the the release. will be incorporated into GAWB-2254

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/methods/MethodConfigDetailsPage.scala (hunk 1 above)
Comment (breilly2) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/methods/MethodConfigDetailsPage.scala:223]: Right, sorry... that was me reviewing from my phone in front of Starbucks on eclipse day. I guess maybe it's the `find(...)` below that you don't technically need. All depends on how you look at it. I'm fine with it either way.

**Pull request closed without merging**